validator/src/main/resources/validations/default-mocked-server-prometheus-validation.yml

```diff
@@ -0,0 +1,4 @@
+-
+  validationType: "mocked-server"
+  # no call to prometheus sample app needed to produce metrics
```

Comment:
i guess this comment should be "no call to prometheus sample app needed to get expected metrics" ?

Reply:
my understanding was that the caller is used to tell the sample app to produce metrics to send to AOC, rather than get expected metrics